app/graphql/types/customers/usage/charge.rb

```diff
@@ -8,13 +8,18 @@ class Charge < Types::BaseObject
 
         field :amount_cents, GraphQL::Types::BigInt, null: false
         field :events_count, Integer, null: false
+        field :id, ID, null: false
```

Comment:
@ansmonjol Can we add the new field to the [spec](https://github.com/getlago/lago-api/blob/main/spec/graphql/types/customers/usage/charge_spec.rb#L8) as well, please?

Reply:
I just added it.

Reply:
Thanks a lot!!